blueprint/index.md

```diff
@@ -1,305 +1,346 @@
-## **Introduction**
-Welcome to the wonderful world of Gig Customer Service – also known as GigCX! This blueprint contains all the components you need to implement Limitless GigCX into your Genesys Cloud organization.
+## Content
+[toc]
```

Comment:
We need to also include some front matter here.  Otherwise, it will not show up in the blueprint section of the developer center.

---
title: Deploy a simple IVR using Terraform, CX as Code, and Archy
author: john.carnell
indextype: blueprint
icon: blueprint
image: images/SimpleIvrFlowDeploy.png
category: 5
summary: |
  This Genesys Cloud Developer Blueprint provides a simple example of how to deploy a touch-tone based IVR using Terraform, CX as Code, and Archy.  
---

Thats an example of the front matter.  It has to be write at the top of the file